pywbem/_subscription_manager.py

```diff
@@ -870,28 +926,29 @@ def add_filter(self, server_id, source_namespaces, query,
 
         # server_id is validated in _create_...() method.
 
-        if filter_id is None and name is None:
-            raise ValueError("The filter_id and name parameters are both "
-                             "None, but exactly one of them must be specified")
-        if filter_id is not None:
+        if owned:
```

Comment:
Same question about owned parameter as for add_destination.

Reply:
see response to previous comment